lms/validation/_base.py

```diff
@@ -83,6 +83,7 @@ def parse(self, *args, **kwargs):
 
         parser = pyramidparser.PyramidParser(
             location=kwargs.pop("location", self.location),
+            unknown=None,
```

Comment:
Maybe a comment here? This is a little impenetrable on the surface of it

Reply:
:+1: added a comment